docs/web/galleryTest.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>Image Display</title>
+        <link rel="stylesheet" type="text/css" href="experimental.css"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta charset="utf-8"/>
+    </head>
+    <body>
+        <div class="grid">
+            <h1>Image Testing</h1>
+            <p>This page is meant to test image display. I rendered all the thumbnails, but on the main archive, they aren't displaying right. The width is always 200 px with landscapes, while the height is 200 px for portraits.</p>
+            <p>I have a feeling that when using the thumbnail property, it tried to keep the aspect ratio of the thumbnails. If that is true, I will need help recoding the gallery. They aren't aligned evenly, and the images hover over the overlays or have them protrude from the sides. Let's see if my hypothesis is right.</p>
+            <p>-Rachel</p>
```

Comment:
I'll be interested to see how the Image Magick thumbnails turned out on your page - I'm sure the alignment stuff is something we could take a look at fixing with your CSS grid!

Reply:
I agree with Mia, I think if you do something with CSS grid that would help fix your alignment issues. So far good work! 👍